### PR TITLE
backport: 'trivial' pr5

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: src/qt/rpcconsole.cpp [hunk 1]
```diff
@@ -574,6 +574,7 @@ bool RPCConsole::eventFilter(QObject* obj, QEvent *event)
             if(obj == autoCompleter->popup()) {
                 autoCompleter->popup()->hide();
                 QApplication::postEvent(ui->lineEdit, new QKeyEvent(*keyevt));
+                autoCompleter->popup()->hide();
```
Comment (UdjinM6) [inline src/qt/rpcconsole.cpp:577]: (14450) this was fixed via https://github.com/dashpay/dash/pull/1898 already. If you want this to be merged purely to align code/history with bitcoin you should drop line 575.

File: test/functional/feature_filelock.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Check that it's not possible to start a second bitcoind instance using the same datadir or wallet."""
```
Comment (UdjinM6) [inline test/functional/feature_filelock.py:5]: (14426) needs dashification here and below

File: src/qt/test/addressbooktests.cpp [hunk 3]
```diff
@@ -136,5 +137,16 @@ void TestAddAddressesToSendBook()
...
 void AddressBookTests::addressBookTests()
 {
+#ifdef Q_OS_MAC
+    if (QApplication::platformName() == "minimal") {
+        // Disable for mac on "minimal" platform to avoid crashes inside the Qt
+        // framework when it tries to look up unimplemented cocoa functions,
+        // and fails to handle returned nulls
+        // (https://bugreports.qt.io/browse/QTBUG-49686).
+        QWARN("Skipping AddressBookTests on mac build with 'minimal' platform set due to Qt bugs. To run AppTests, invoke "
+              "with 'test_bitcoin-qt -platform cocoa' on mac, or else use a linux or windows build.");
```
Comment (UdjinM6) [inline src/qt/test/addressbooktests.cpp:147]: (14011) needs dashification

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Comment by PastaPastaPasta** [comment]
Done

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**